### DISCUSSION TITLE
How to customize the path of models

### DISCUSSION THREAD

**Discussion by luoxr18** [cubiq/ComfyUI_IPAdapter_plus#694] [Q&A]
Hi, I want to customize the path of the ip-adapter models, how can i modify the script?

![image](https://github.com/user-attachments/assets/5648f2ee-b699-4d82-954b-6497ae7e6d05)

**Comment by airtonix** [comment]
https://github.com/cubiq/ComfyUI_IPAdapter_plus/blob/main/README.md?plain=1#L82
